src/reducers/channels.js

```diff
@@ -150,6 +150,21 @@ export default function reduce(state = initialState, action) {
       return newState
     }
 
+    case types.UPDATE_CHANNEL_UNREAD_COUNTER: {
+      const { id, unread, time } = action.payload
+      const index = findIndex(state, { id })
+      if (index === -1) return state
+      const channel = state[index]
+      if (channel.current) return state
```

Comment:
@AleshaOleg why do we return the original state if it's the current channel?

I was thinking: If i scrolled to old messages I still might have unread messages and the counter should be updated?